Exec/EB_cases/EB_FlowPastCylinder/inputs.2d-regt

```diff
@@ -202,3 +204,16 @@ nodal_proj.atol =  1.0e-12
 nodal_proj.verbose = 1
 #nodal_proj.maxiter = 500
 #ns.plot_centroid_data = false 
+
+ns.refine_cutcells = 1
+
```

Comment:
This HAS to be the default right now, since we don't have code in place for re-redistribution, correct?